src/librustc/session/config.rs

```diff
@@ -507,6 +507,8 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
         "system linker to link outputs with"),
     link_args: Option<Vec<String>> = (None, parse_opt_list,
         "extra arguments to pass to the linker (space separated)"),
+    link_dead_code: bool = (false, parse_bool,
+        "let the linker strip dead coded (turning it on can be used for code coverage)"),
```

Comment:
this description is not right, seems it should be `"let the linker **link** dead code"`